### ISSUE TITLE
Is there a way to get notified when new netns is created

### ISSUE THREAD

**Issue by msherif1234** [vishvananda/netns#77]
my application when it starts it detect all available netns by looking at `/var/run/netns` but while its running if new netns is created my application won't know anything about it and as a result it won't know about links and their attributes 

what is the best way to know runtime when netns is created so we can use `LinkSubscribeAt()` for it ?

**Comment by bn222** [comment]
I believe we can use inotify to watch for creation/deletion events in `/var/run/netns`

**Comment by msherif1234** [comment]
Yeah I added inotify and when netns is created I subscribe for notificatoin using `netlink.LinkSubscribeAt` but it failed [here](https://github.com/vishvananda/netlink/blob/6afddb37c1f00693528264222a57a3c08dccbab7/nl/nl_linux.go#L703)

**Comment by msherif1234** [comment]
cc'd @vishvananda
